src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties' :
+        Object.entries(action.extraData).forEach(([key, value]) => {
+          state[key] = value;
+        });
+        break;
+
+      case 'clear' :
+        Object.keys(state).forEach(key => {
+          delete state[key];
+        });
+        break;
+
+      case 'removeProperties' :
+        action.keysToRemove.forEach(key => {
+          delete state[key];
+        });
+        break;
```

Comment:
![image](https://github.com/mate-academy/js_stateful-object/assets/96409563/9402e2cf-f801-4382-b0d4-ed1fb4a530e7)
Default case is missing